### ISSUE TITLE
Perpetual Ore

### ISSUE THREAD

**Issue by NekoDaKat** [tgstation/-tg-station#9897]
So, the Ore is breaking into the items like usual, but the blocks are staying behind, causing a perpetual ore paradox, and that's no good.

meme

**Comment by SubtleGraces** [comment]
Yep, had somebody else mention this on Sybil.

Ore deposits are essentially infinite right now.

**Comment by Boggart** [comment]
There's a runtime with ore tiles and logging that prevents them from changing to the airless mining turf.

```
runtime error: Cannot read /obj/item/weapon/ore/silver (/obj/item/weapon/ore/silver).name
proc name: gets drilled (/turf/simulated/mineral/proc/gets_drilled)
source file: mine_turfs.dm,452
```

**Comment by xxalpha** [comment]
Checking for name of type.

**Comment by Iamgoofball** [comment]
Fuck, seriously? Wow

Will fix

**Comment by Lumipharon** [comment]
materials are functionally infinite now anyway.

Make a crusher in R&D, chuck random shit in that costs 1 metal each, get x4 of any kind of mineral back everytime (might not be every time, but if it isn't, it's damn close.)

**Comment by francinum** [comment]
@Lumipharon with t4 parts.

**Comment by Lumipharon** [comment]
t4 is not hard to get.

**Comment by phil235** [comment]
fixed by #9909
